.tests/models/adult/model/serving_test.yaml

```diff
@@ -0,0 +1,72 @@
+---
+contract:
+  inputs:
+    age:
+      profile: numerical
+      shape: scalar
+      type: int64
+    capital_gain:
+      profile: numerical
+      shape: scalar
+      type: int64
+    capital_loss:
+      profile: numerical
+      shape: scalar
+      type: int64
+    country:
+      profile: numerical
+      shape: scalar
+      type: int64
+    education:
+      profile: numerical
+      shape: scalar
+      type: int64
+    hours_per_week:
+      profile: numerical
+      shape: scalar
+      type: int64
+    marital_status:
+      profile: numerical
+      shape: scalar
+      type: int64
+    occupation:
+      profile: numerical
+      shape: scalar
+      type: int64
+    race:
+      profile: numerical
+      shape: scalar
+      type: int64
+    relationship:
+      profile: numerical
+      shape: scalar
+      type: int64
+    sex:
+      profile: numerical
+      shape: scalar
+      type: int64
+    workclass:
+      profile: numerical
+      shape: scalar
+      type: int64
+  name: predict
+  outputs:
+    classes:
+      profile: numerical
+      shape: scalar
+      type: int64
+install-command: pip install -r requirements.txt
+kind: Model
+monitoring:
+- config:
+    monitoring-model: adult_monitoring_test:27
```

Comment:
Specifying monitoring config here to a specific model version can cause tests to fail if some previous tests were failed. Can we omit it here?